src/video/dummy/SDL_nullvideo.c

```diff
@@ -55,6 +55,17 @@
 static int DUMMY_VideoInit(SDL_VideoDevice *_this);
 static void DUMMY_VideoQuit(SDL_VideoDevice *_this);
 
+static int DUMMY_SetWindowPosition(SDL_VideoDevice *_this, SDL_Window *window)
+{
+    SDL_SendWindowEvent(window, SDL_EVENT_WINDOW_MOVED, window->floating.x, window->floating.y);
```

Comment:
Should this be the default implementation if the backend doesn't have SetWindowPosition/SetWindowSize functions?

Reply:
Actually, thinking it through, probably not. We want to reflect that the window doesn't move and isn't resized in fullscreen implementations like KMSDRM, Android, etc.

Reply:
That was my reasoning behind it as well, especially on lower-level platforms where the 'window' is just the screen.